CONTRIBUTING.md

```diff
@@ -147,6 +147,7 @@ started you should:
 1. fork the `grommet` repository
 1. clone it `git clone https://github.com/<your-username>/grommet.git`
 1. install dependencies using: `yarn install`
+1. start running the storybook locally using: `yarn storybook`
```

Comment:
I don't think you need to run a storybook in order to file a PR. Can you please share your thoughts on why you think it is needed?

Reply:
Yes, we don't need to run a storybook in order to file a PR. But when we make any changes locally there is no way we know how to run the storybook. It can be optional step before going and submitting the pr. I didn't know how to run locally until you mentioned on one of my PR review.so I thought it would be useful new contributers/developer coming, When trying to make changes locally.

Reply:
Gotcha. It is detailed in the README.md page https://github.com/grommet/grommet#explore, but I might agree it should be mentioned on the Contribution page as well.
Would you be able to add a section for testing your code and explain it there?

Reply:
ok sure, will give it a try.

Reply:
@ShimiSun  added as a seperate section

Reply:
Great thanks for the follow up!